en/docs/Learn/APIGateway/api-gateways-with-dedicated-backends.md

```diff
@@ -4,19 +4,19 @@ We can extend the [multiple gateway environments](../maintaining-separate-produc
 
 However, the API Publisher can only provide a single static endpoint for an API in the implementation. Therefore, the API call is directed to a single endpoint in whichever Gateway the API is deployed in, as depicted in the diagram below.
 
-[![]({../../../../assets/img/Learn/single-endpoint.png)](../../../../assets/img/Learn/single-endpoint.png)
+[![]({../../../../assets/img/Learn/single-endpoint.png)]({../../../../assets/img/Learn/single-endpoint.png)
```

Comment:
@Arshardh , can you run the `mkdocs build` command and see if there are any errors that correspond to your file. I have a feeling that your image path should be ../../assets/img/Learn/